nmpolicy/internal/parser/parser.go

```diff
@@ -158,6 +162,18 @@ func (p *parser) parseNumber() error {
 	return nil
 }
 
+func (p *parser) parseBoolean() error {
+	boolean, err := strconv.ParseBool(p.currentToken().Literal)
```

Comment:
use token.isTrue, isFalse we know that lexer is going to allow true/false but let's do the same here just in case.